docs/source/examples/notebooks/creating_models/6-a-simple-SEI-model.ipynb

```diff
@@ -576,7 +580,7 @@
     "    ax1.set_ylabel(r'SEI thickness [$\\mu$m]')\n",
     "    ax1.set_xlabel(r't [$\\mu$s]')  \n",
     "    \n",
-    "    plot_c, = ax2.plot(x * 1e6 * L_out(t_in_seconds), c_out(t_in_seconds, x_in_metres))\n",
+    "    plot_c, = ax2.plot(x * 1e6 * L_out(t_in_seconds), c_out(t_in_seconds, x_in_metres))     # noqa: F821\n",
```

Comment:
This is an actual error and should not be ignored. It looks like `x_in_metres` is not defined in the notebook anywhere and `ruff` caught that. @brosaplanella might be able to help with this better 🙂

Reply:
That is really weird! Locally the notebook doesn't run (both @arjxn-py and I checked) and obviously it shouldn't, as we are calling an undefined variable. @tinosulzer @rtimms do you have any ideas?